systems/WikibaseRepo/05-Building_Block_View.md

```diff
@@ -6,7 +6,38 @@ TBA
 
 ## External Interfaces
```

Comment:
I think we should get rid of this "External interfaces" heading and instead just had "Action API" as a 2nd level heading.
This will mean that it appears in the side bar for example.
This would be consistent with https://github.com/wmde/wikidata-wikibase-architecture/blob/main/systems/WikibaseClient/05-Building_Block_View.md that has both the overal heading and then the top level blocks as level 2 headings

Reply:
done